libparsec/crates/platform_mountpoint/src/unix/filesystem.rs

```diff
@@ -1086,6 +1169,17 @@ impl fuser::Filesystem for Filesystem {
             offset,
             data.len(),
         );
+        let log = std::path::PathBuf::from(format!(
+            "/home/emmanuel/projects/parsec-cloud/fs_{ino:x}_{offset:x}.log"
+        ));
+        std::fs::File::options()
+            .create(true)
+            .append(true)
+            .write(true)
+            .open(log)
+            .unwrap()
+            .write_all(data)
```

Comment:
Debug artefacts?

Reply:
No, it's a low tech backdoor :ninja: :trollface: